### ISSUE TITLE
Paper #656

### ISSUE THREAD

**Issue by kstolee** [researchart/fse20#113]
This issue is for discussion related to PR #15 

Author contact: @SteveZhangBit

**Comment by kstolee** [comment]
# Authors
## Changjian Zhang

Carnegie Mellon University

Email: changjiz@andrew.cmu.edu

GitHub: https://github.com/SteveZhangBit

## David Garlan

Carnegie Mellon University

Email: garlan@cs.cmu.edu

GitHub:

## Eunsuk Kang (Corresponding Author)

Carnegie Mellon University

Email: eunsukk@andrew.cmu.edu

GitHub:

**Comment by art-reviewer** [comment]
[Paper](https://github.com/SteveZhangBit/fse20/blob/6da37f23a88c4f2b38e51b006eb8cd43c85bd716/submissions/reusable/FSE-656/esecfse2020-paper656.pdf)

**Comment by anon-review** [comment]
### Summary

- [x]  Reusable

### Evaluation

**Functional**

- [x]  Documented
- Inventory of artefacts included along with descriptions (*robustness-calculator.jar* and *models.zip*)
- Required Java version specified, and I was able to get to the expected help text easily
- CLI tool's help text includes documentation for the accepted arguments
- Documentation for system description files also included, so one can run their own evaluations using this implementation

- [x]  Consistent

Contains the implementation described in the paper and models for the presented case studies.

- [x]  Complete

Artefact appears to be complete. Models and system description files are included for each stage in each case study.

- [x]  Exercisable

Artefact is exercisable following the instructions and examples provided by the authors. Results from section 7 were reproduced in the resulting JSON files using the `-o` argument

**Reusable**

Artefact is of high quality. I especially appreciated the following:

- pointers to relevant sections and tables in the paper, so I knew exactly how to reproduce specific results
- documentation includes instructions not only to *run* the tool, but also to *build* and *configure* the tool, allowing reuse and repurposing

**Comment by gousiosg** [comment]
@a-hopefully-friendly-reviewer We would appreciate your review till Monday morning AoE.

**Comment by hopefully-anon** [comment]
✅ I find this artifact to be **Resusable**  (fulfills all requirements for functional and is especially well-documented)

🟡 It appears that the authors are ***not*** applying for the **Available** badge and I could not find a DOI link.

Summary:
1. **Installation**  ✅ : Software was straightforward to download and run.
2. **Public Availability** : 
3. **Documentation** ✅ : Explains the contents of the artifact, the dependencies required to run the `.jar` files, and identifies links between the artifact and corresponding sections of the paper (thank you).
4. **Consistent/Complete** ✅: Results of running the tool match the expected results identifies in the `README.md` file.
5. **Exercisable** ✅: Program runs as expected.  
6. **Reusability** ✅ : Also includes documentation on how to compile and run the program, making it easier to modify/repurpose/reuse the code.

**Comment by gousiosg** [comment]
To the authors: would you also like to apply for the _available_ badge? If yes, please update your artifact with the appropriate documentation and a DOI.

**Comment by SteveZhangBit** [comment]
Yes. I guess I misunderstood the requirements that if we apply for the _reusable_ badge, we don't have to do the DOI thing. We will update the documentation and the DOI very soon. Thank you very much!

**Comment by kstolee** [comment]
Looks like the DOI was added in PR #151 

@hopefully-anon and @anon-review can you verify that ```available``` is appropriate?

**Comment by anon-review** [comment]
@kstolee The provided DOI was **not** found by [doi.org](https://doi.org).

**Comment by SteveZhangBit** [comment]
I'm using figshare. It seems that to make it accessible through DOI, it has to go through some review process. Right now, it is under the review of my organization.

**Comment by kstolee** [comment]
@SteveZhangBit please ping us here when it is available.

**Comment by SteveZhangBit** [comment]
@kstolee Right now, we don't know how long it would take for the internal review process of our organization. If it is emergent, we would contact the administrators, or we could publish it by using another service. Do we have a deadline for this? Thanks!

**Comment by kstolee** [comment]
@SteveZhangBit badge decisions will be made by July 1. We'd appreciate the artifact to be available a day in advance.

**Comment by SteveZhangBit** [comment]
@kstolee I believe the link should work now. Thank you!

**Comment by kstolee** [comment]
Great @SteveZhangBit ! 

I'm applying ```reusable``` and ```functional``` to start. 

@hopefully-anon and @anon-review can you please confirm whether or not you can now recommend ```available```?

**Comment by hopefully-anon** [comment]
`available` confirmed by testing DOI link  [http://doi.org/10.1184/R1/12543308](http://doi.org/10.1184/R1/12543308)

**Comment by anon-review** [comment]
`available` confirmed. Congrats to @SteveZhangBit et al.

**Comment by gousiosg** [comment]
Great work everybody!

**Comment by SteveZhangBit** [comment]
Thank you ALL!
